README.md

```diff
@@ -1 +1,19 @@
-# se701-assignment1
+# Threader
```

Comment:
Is this what we're calling it?

Reply:
We'll find out in 11pm when voting closes :ballot_box:

Reply:
I'll edit the logo based on the result

Reply:
Threader it is